Patch to fix missing link libraries in pkgconfig
for correctly linking with external Lua.
author: Arnold H. Kole (Utrecht University)
adapted to version 0.8.4 by Miguel Dias Costa (University of Coimbra)
--- flook.pc.in.orig	2025-02-13 15:52:24.022161421 +0000
+++ flook.pc.in	2025-02-13 15:52:53.402133931 +0000
@@ -9,5 +9,5 @@
 Version: @PROJECT_VERSION@
 URL: https://github.com/ElectronicStructureLibrary/flook
 Cflags: -I${includedir}
-Libs: -L${libdir} -lflookall -ldl
+Libs: -L${libdir} -lflookall -llua -ldl

